mmaction/core/evaluation/eval_hooks.py

```diff
@@ -40,6 +39,8 @@ def __init__(self, *args, save_best='auto', **kwargs):
 
 if not from_mmcv:
 
+    from mmcv.runner import Hook
```

Comment:
There should be many imported packages that are only used in local code blocks. Do you want to modify them all?

Reply:
Some of them may cause circle import, so just keep it.